tests/console/repo_package_install.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check SLE repositories, packages and installation status
+# Maintainer: Zaoliang Luo <zluo@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+my @array = (
```

Comment:
Bad choice of name, especially because you actually treat it like a hash

Reply:
Ok, not a hash, an array of hashes but still ... how about @packages?

Reply:
okay, then use @packages for array, change pack to name